src/generated/Probabilistic.jl

```diff
@@ -6,7 +6,7 @@ This file is auto-generated. Do not edit.
         name::String
         resolution::Dates.Period
         percentiles::Vector{Float64}
-        data::SortedDict{Dates.DateTime, Array}
+        data::Union{SortedDict{Dates.DateTime, Matrix{CONSTANT}}, SortedDict{Dates.DateTime, Matrix{POLYNOMIAL}}, SortedDict{Dates.DateTime, Matrix{PWL}}}
```

Comment:
@jd-lara I made comparable changes for Probabilistic.  I suggest that we leave this PR open until I get a chance to fix Scenarios as well.

Reply:
That's ok, we should target wrapping up Monday though to focus on the pending items in PSY.

Reply:
Also, I suspect that deserializing Probabilistic and Scenarios are broken with InMemoryTimeSeriesStorage.  There are no tests for it.  I'll address this.

Reply:
@jd-lara I made comparable changes for Scenarios.  I did not make any changes for InMemoryTimeSeriesStorage.  It seems that you did not implement the required deserialize_time_series methods for that struct when you added Probabilistic and Scenarios.  I'll leave this to you.